pkgs/tools/networking/dd-agent/default.nix

```diff
@@ -26,61 +33,158 @@ let
     # due to flake8
     doCheck = false;
   };
+  version = "5.13.2";
+
+  integrations = fetchFromGitHub {
+    owner = "datadog";
+    repo = "integrations-core";
+    rev = version;
+    sha256 = "1nbjmkq0wdfndmx0qap69h2rkwkkb0632j87h9d3j99bykyav3y3";
+  };
+
+  jmxArtifact = fetchurl
+    { url = "http://dd-jmxfetch.s3.amazonaws.com/jmxfetch-0.13.0-jar-with-dependencies.jar";
+      sha256 = "02rfn17q42kxiv2sv306ibdhvi8l4sbjdwhh1nwdlxw46pqh4ks1";
+    };
 
 in stdenv.mkDerivation rec {
-  version = "5.11.2";
   name = "dd-agent-${version}";
 
   src = fetchFromGitHub {
     owner  = "datadog";
     repo   = "dd-agent";
     rev    = version;
-    sha256 = "1iqxvgpsqibqw3vk79158l2pnb6y4pjhjp2d6724lm5rpz4825lx";
+    sha256 = "0x2bxi70l2yf0wi232qksvcscjdpjg8l7dmgg1286vqryyfazfjb";
   };
 
   buildInputs = [
     python
     unzip
     makeWrapper
-    pythonPackages.requests
-    pythonPackages.psycopg2
-    pythonPackages.psutil
+    pythonPackages.boto
+    docker_1_10
+    pythonPackages.kazoo
     pythonPackages.ntplib
-    pythonPackages.simplejson
-    pythonPackages.pyyaml
-    pythonPackages.pymongo_2_9_1
-    pythonPackages.python-etcd
     pythonPackages.consul
-    docker_1_10
-  ];
-  propagatedBuildInputs = with pythonPackages; [ python tornado ];
+    pythonPackages.python-etcd
+    pythonPackages.pyyaml
+    pythonPackages.requests
+    pythonPackages.simplejson
+    pythonPackages.supervisor
+    pythonPackages.tornado_3_2_2
+    pythonPackages.uptime
+  ] ++ extraBuildInputs;
```

Comment:
Since this PR seems to be the direction dd-agent is going for updates, I wanted to let you know that I discovered a problem with the existing package which could be fixed here.

dd-agent has proxy functionality which is currently not working because it relies on pycurl, which is not in the package dependencies.  I've patched it for my install, but official support by adding the pycurl dependency here would fix a bug in the current package.